tap_zoom/client.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, config, config_path):
         self.__use_jwt = False
         self.__expires_at = None
         self.start_date = config.get('start_date')
+        """
+            Add the `sync_today` flag if you want to also ingest data as it's coming
+            in for the current day. 
+            
+            NOTE: Since the Zoom API only takes dates (instead of timestamps) for the 
+            date ranges when requesting recordings, the share_url for the current 
+            dates are not stable. Therefore make sure to also use the url_to_recordings
+            to map share_url to a stable uuid: https://github.com/Pathlight/tap-zoom/pull/5
+        """
+        self.sync_today = config.get('sync_today', False)
```

Comment:
this should default to true to not disrupt existing tap behaviour, right? previously by default it would sync for today. this option is to opt out of syncing for today?

Reply:
Sorry not clear: only version `1.0.0` is currently deployed, which did not have the behavior to sync yet. We didn't deploy the changes (`1.0.1` commit [here](https://github.com/Pathlight/tap-zoom/commit/7b2d9bd3ca3e9fd8bb28ad83d1eb6952aefada7b)) since we deprioritized improving the zoom tap for our internal Pathlight CI instance. So the default behavior right now in production is that `sync_today` is False

Reply:
Ahh got it. 1.0.0 (currently active) doesn't sync today which is the default behaviour. so in 1.0.2 (this version), we can opt-in to sync today. 1.0.1 previously always included today which was problematic and we reverted to 1.0.0

Reply:
Yup exactly! I'll bump the version to `1.0.2` as well just so we make sure not to miss this sync flag change